CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and `a-btn__icon-on-right` classes referenced in docs to correct
 `a-btn_icon__on-left` and `a-btn_icon__on-right` classes.
 - **cf-forms:** [PATCH] Changed label heading from 5 to 4, per design manual.
 - **cf-typography:** [PATCH] Remove bottom margin from `m-slug-header`.
+- **capital-framework:** Updates `gulp-autoprefixer` to `4.0.0`.
+- **capital-framework:** Updates `gulp-uglify` to `3.0.0`.
```

Comment:
Fixed in https://github.com/cfpb/capital-framework/pull/521/commits/fc4abaa85862ed478eedd36033e1556cc11775a1